vm/vm.go

```diff
@@ -278,6 +278,7 @@ func (vm *VM) initConstants() {
 	vm.objectClass.constants["STDIN"] = &Pointer{Target: vm.initFileObject(os.Stdin)}
 }
 
+// TopLevelClass returns a specified class or a top level class
```

Comment:
It returns a specified top-level class (stored under the `Object` constant, like `String`, `Integer`..etc.)

Reply:
Updated